src/styles/components/_hawk-facet-rail.scss

```diff
@@ -271,6 +271,10 @@
 	display: block
 }
 
+.hawk-result-rail__keyword-not-match {
+	color:#ff0000;
```

Comment:
isn't there like a `$red` or `$error` color on the palette? (curiosity, not blocker)

Reply:
This was the best I could do close to the color shown. I couldn't find `$red` or `$error`